site/content/deployment/kubernetes.md

```diff
@@ -45,6 +45,71 @@ From the directory `deploy/target/deploy/kubernetes` run:
 ~hono/deploy/target/deploy/kubernetes$ ./olm_deploy.sh
 ~~~
 
+## Helm deployment
+
+The easiest way to deploy Eclipse Hono to Kubernetes cluster is to use Helm. You can use Helm with or without Helm Tiller server. In order to use Helm, you need to have it installed properly on your system.
+Please follow [these instructions](https://docs.helm.sh/install/) on how to install Helm tool locally and optionally Tiller server.
+
+### Install Eclipse Hono with Helm
+
+To deploy Eclipse Hono to the cluster with installed Tiller server, simply run
+
+~~~sh
+~hono/deploy$ helm install --dep-up --name eclipse-hono --namespace hono target/deploy/helm/
+~~~
+
+This will create a new `hono` namespace in the cluster and install all the components. The name of the Helm release will be `eclipse-hono`.
+
+You can check the status of the deployment with one of the following commands
+
+~~~sh
+$ helm list
+$ helm status eclipse-hono
+$ helm get eclipse-hono
+~~~
+
+### Install Eclipse Hono without using Helm Tiller server
+
+If you for whatever reason can't or don't want to install Helm Tiller server in your cluster, you can still use the resources and deploy them manually.
+To generate Eclipse Hono resources locally with Helm, run
+
+~~~sh
+~hono/deploy$ helm dep update target/deploy/helm/
+~hono/deploy$ helm template --name eclipse-hono --namespace hono --output-dir . target/deploy/helm/
+~~~
+
+This should create `eclipse-hono` folder with all the resources. Now, you can use `kubectl` to deploy them to any Kubernetes cluster
+
+~~~sh
+~hono/deploy$ kubectl apply -f ./eclipse-hono --namespace hono
+~~~
+
+### Use Eclipse Hono
+
+After successful installation, you can proceed and [access your Hono services](#accessing-hono-services)
+
+Note that in this deployment default Grafana credentials are (admin:prom-operator)
+
+### Uninstall Eclipse Hono
+
+To uninstall Eclipse Hono using Helm tiller server, run
+
+~~~sh
+~hono/deploy$ helm delete --purge eclipse-hono
+~hono/deploy$ kubectl delete crd prometheuses.monitoring.coreos.com
+~hono/deploy$ kubectl delete crd prometheusrules.monitoring.coreos.com
+~hono/deploy$ kubectl delete crd servicemonitors.monitoring.coreos.com
+~hono/deploy$ kubectl delete crd alertmanagers.monitoring.coreos.com
+~~~
+
+The additional delete commands are necessary to clean [Prometheus operator CRDs](https://github.com/helm/charts/tree/master/stable/prometheus-operator#uninstalling-the-chart).
```

Comment:
Are the CRDs related to Prometheus only? Or do they belong to OLM?

Reply:
They are related to Prometheus operator only. OLM is not mandatory for the Helm deployment.

Reply:
So, in this case you are deploying Prometheus *without* OLM, right?

Reply:
That is my understanding, yes. It's uses only Prometheus Operator for now. We can see how to approach all this in the future.